### PR TITLE
Create nersc_pipeline.yml

### PR DESCRIPTION
**Pull request by adityakavalur**
Need to trigger Luke's E4S builds on Cori for a commit to Spack

### CODE REVIEW

**Comment by adityakavalur** [comment]
@scottwittenburg : @eugeneswalker , @shahzebsiddiqui , @paulbry and I created a pipeline trigger similar to the Ascent pipeline to test E4S builds on Cori

File: share/spack/gitlab/nersc_pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+merge_pipeline:
+  only:
+  - develop
+  variables:
+    SPACK_REPO: ${CI_PROJECT_URL}
+    SPACK_REF: ${CI_COMMIT_SHA}
+  trigger:
+    project: ecp/e4s/e4spipelines/e4s-luke
```
Comment (scottwittenburg) [inline share/spack/gitlab/nersc_pipeline.yml:8]: I haven't verified that this is the right project path.  Seems like a lot of slashes, but maybe that's how it's set up?  Out of curiosity, is the organization `ecp` and the rest is the project?

**Review by scottwittenburg — APPROVED** [review]
This looks fine to me, as it's based on the working job configured at OLCF.

File: share/spack/gitlab/nersc_pipeline.yml (hunk 1 above)
Comment (adityakavalur) [reply share/spack/gitlab/nersc_pipeline.yml:8]: @scottwittenburg : Thanks! I checked the path and file with my fork and it is working as expected. ECP and E4S are groups everything else is project. We are hoping to keep ECP teams within some sort of structure

**Comment by adityakavalur** [comment]
Closing this as it is a duplicate of #19688

**Comment by shahzebsiddiqui** [comment]
this is a duplicate PR see one from https://github.com/spack/spack/pull/19688. This is WIP but we have a multi-project pipeline setup i will communicate in other PR

**Pull request closed without merging**